tests/test_modeling_physical_validation.py

```diff
@@ -0,0 +1,161 @@
+                                                 Om0=TEST_CASE['cosmo_Om0'],
+                                                 Ob0=TEST_CASE['cosmo_Ob0'])
+COSMO_CCL = clmm.cclify_astropy_cosmo(COSMO_APY)
+
+# Sets of parameters to be used by multiple functions
+RHO_PARAMS = {
+    'mdelta':TEST_CASE['cluster_mass'],
+    'cdelta':TEST_CASE['cluster_concentration'],
+    'z_cl':TEST_CASE['z_cluster'],
+    'cosmo':COSMO_CCL,
+    }
+SIGMA_PARAMS = {
+    'mdelta':TEST_CASE['cluster_mass'],
+    'cdelta':TEST_CASE['cluster_concentration'],
+    'z_cl':TEST_CASE['z_cluster'],
+    'cosmo':COSMO_CCL,
+    'delta_mdef':TEST_CASE['mass_Delta'],
+    'halo_profile_model':TEST_CASE['density_profile_parametrization'],
+    }
+GAMMA_PARAMS = {
+    'mdelta':TEST_CASE['cluster_mass'],
+    'cdelta':TEST_CASE['cluster_concentration'],
+    'z_cluster':TEST_CASE['z_cluster'],
+    'z_source':TEST_CASE['z_source'],
+    'cosmo':COSMO_CCL,
+    'delta_mdef':TEST_CASE['mass_Delta'],
+    'halo_profile_model':TEST_CASE['density_profile_parametrization'],
+    'z_src_model':'single_plane',
+    }
+
+def test_physical_constants():
+    '''
+    Test physical values of physical_constants
+
+    Notes
+    -----
+        The precision set for these tests are somewhat arbitrary
+        has to be better defined at some point
+    '''
+    tst.assert_allclose(TEST_CASE['lightspeed[km/s]'], clmmconst.CLIGHT_KMS.value, 1e-3)
+    tst.assert_allclose(TEST_CASE['G[m3/km.s2]'], clmmconst.GNEWT.value, 1e-3)
+    tst.assert_allclose(TEST_CASE['pc_to_m'], clmmconst.PC_TO_METER.value, 1e-6)
+    tst.assert_allclose(TEST_CASE['Msun[kg]'], clmmconst.SOLAR_MASS.value, 1e-2)
+    tst.assert_allclose(TEST_CASE_SIGMAC_PHYSICAL_CONSTANT, CLMM_SIGMAC_PHYSICAL_CONSTANT, 1e-2)
+
+def test_rho():
+    '''
+    Test physical values of rho
+    '''
+    rho = clmm.get_3d_density(R3D, **RHO_PARAMS)
+    tst.assert_allclose(NC_PROF['rho'], rho*G_PHYSCONST_CORRECTION, 1e-11)
+
+def test_sigma():
+    '''
+    Test physical values of sigma
+    '''
+    tst.assert_allclose(NC_PROF['Sigma'], G_PHYSCONST_CORRECTION*\
+                        clmm.predict_surface_density(R3D, **SIGMA_PARAMS), 1e-9)
+
+def test_delta_sigma():
+    '''
+    Test physical values of delta_sigma
+    '''
+    tst.assert_allclose(NC_PROF['DeltaSigma'], G_PHYSCONST_CORRECTION*\
+                        clmm.predict_excess_surface_density(R3D, **SIGMA_PARAMS), 1e-8)
+
+def test_get_da():
+    '''
+    Test physical values of Da
+    '''
+    dl_clmm = clmm.modeling.get_angular_diameter_distance_a(COSMO_CCL,
+                                                            TEST_CASE['aexp_cluster'])
+    ds_clmm = clmm.modeling.get_angular_diameter_distance_a(COSMO_CCL,
+                                                            TEST_CASE['aexp_source'])
+    dsl_clmm = clmm.modeling.get_angular_diameter_distance_a(COSMO_CCL,
+                                                             TEST_CASE['aexp_source'],
```

Comment:
When the import scheme is changed above (`from clmm.modeling import get_angular_diameter_distance_a, otherfunc2, otherfunc3`) this line will be much shorter and we should be able to get the assert statements on the same lines as the computations.

Reply:
We want this because if a test fails, we want the error message to include the function call that failed, not just `dl_clmm`

Reply:
fixed